pkg/rancher-desktop/main/mainmenu.ts

```diff
@@ -57,6 +57,7 @@ function getHelpMenu(isMac: boolean): MenuItem {
         role:  'about',
         label: `&About ${ Electron.app.name }`,
         click() {
+          Electron.app.setAboutPanelOptions({"applicationName": "Rancher Desktop by SUSE"});
```

Comment:
Instead of setting this here, let's update `setAboutPanelOptions` in `background.ts`